assets/javascripts/birdeck.js

```diff
@@ -60,37 +60,26 @@ function createPost() {
   $("#create-post").on("click", function() {
     var postParams = {
       post: {
-        description: $("#post-description").val()
```

Comment:
@stevekinney Like the use of the initialized variables. Don't like the use of a higher level abstraction for the AJAX call just now as I mentioned in one of my previous comments. I think understanding AJAX at this point for them is hard enough, and I rather have them understand the raw `$.ajax()` method without the higher level functions and the promises. They can get to that point later in Module 4, IMO.